packages/react-components/react-datepicker/src/components/CalendarDayGrid/CalendarDayGrid.tsx

```diff
@@ -127,53 +127,49 @@ export const CalendarDayGrid: React.FunctionComponent<CalendarDayGridProps> = pr
     getRefsFromDayInfos,
   } as const;
 
-  const arrowNavigationAttributes = useArrowNavigationGroup({
-    axis: 'both',
-    ignoreDefaultKeydown: { Enter: true, Escape: true, Tab: true },
-  });
+  const arrowNavigationAttributes = useArrowNavigationGroup({ axis: 'grid' });
 
   return (
-    <div {...arrowNavigationAttributes}>
```

Comment:
This is a minor change but gh takes it as bigger than it is 😅 just removing the wrapping div and adding the arrow navigation directly to the table.